woof-code/rootfs-skeleton/etc/rc.d/rc.sysinit

```diff
@@ -455,39 +455,44 @@ if [ $PUPMODE -eq 2 ] ; then #full hd installation.
 fi
 
 [ ! -d /proc/acpi ] && modprobe apm #load apm.ko if no acpi.
-zcat /proc/config.gz | grep -m 1 -q '^CONFIG_INPUT_LEDS=m' && modprobe input-leds #for >= k4.2x
+modprobe input-leds 2>/dev/null #for >= k4.2x
 
-### /var/local/xorg_udev is created when:
+### /var/local/xorg_udev is created when X.Org is present and:
 ### - a touchscreen has been detected
 ### - missing xorg input-kbd and input-mouse drivers [ex: ubuntu xenial]
 TOUCHMOD=`lsmod|grep touch`
 ISITANXO=`cat /sys/class/dmi/id/product_name` #mavrothal: Check if it is an OLPC XO
 ISITAMAC=`grep -i -m 1 '^Apple' /sys/class/dmi/id/board_vendor`
-[ -f /lib64/libc.so.6 ] && lsuffix=64 || lsuffix= #slackware64
-if [ ! -f /usr/lib${lsuffix}/xorg/modules/input/kbd_drv.so -a ! -f /usr/lib${lsuffix}/xorg/modules/input/mouse_drv.so ] ; then
-  echo "XORG_UDEV=1: Xorg kbd_drv.so and mouse_drv.so are missing"
-  XORG_UDEV=1
-fi
-
-TOUCHSCREEN="$(udevadm info --export-db | grep -m 1 ID_INPUT_TOUCHSCREEN=1)"
```

Comment:
This check is slow, over 150ms on a relatively fast laptop. Why do it if `XORG_UDEV=1` already?